### ISSUE TITLE
Kedro-airflow setup.py requires test_requirements.txt, but it's not packaged in pypi

### ISSUE THREAD

**Issue by cshaley** [kedro-org/kedro-plugins#53]
## Description
Kedro-airflow setup.py requires test_requirements.txt, but it's not packaged in pypi

## Context
I am trying to package kedro-airflow and kedro-viz for conda-forge so that I can conda install them.  Conda-forge runs setup.py as part of the build process.  Kedro-airflow directly references test_requirements.txt in setup.py, but test_requirements.txt is not packaged in the build for pypi, so running setup.py fails.

## Steps to Reproduce
1. Download kedro-airflow via pypi.
2. unzip it and run python setup.py.  

## Expected Result
Setup.py succeeds.

## Actual Result
Setup.py fails because test_requirements.txt is missing.

```
 FileNotFoundError: [Errno 2] No such file or directory: 'test_requirements.txt'
```

**Comment by deepyaman** [comment]
@cshaley Thanks for the report! My opinion is that `test_requires` and the associated `python setup.py test` are both pretty useless/deprecated, and we don't use that in the CI anyway (thankfully), so the easiest solution is to just kill them. :)

Deprecation references:

* https://github.com/pypa/setuptools/issues/1684
* https://github.com/pytest-dev/pytest/issues/5534

**Comment by cshaley** [comment]
Note: same thing is true of Kedro-viz. I'm also looking to package it on conda forge but this issue got in the way.   @deepyaman Shall I open an issue there as well?

**Comment by cshaley** [comment]
Resolved by #55
